.github/workflows/ci.yml

```
@@ -0,0 +1,34 @@
+name: ci
+
+on:
+  pull_request:
+  push:
+    branches: master
+
+jobs:
+  test:
+    name: test w/ Python ${{ matrix.python-version }}
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
```

I recommend this, it means if one job fails, it allows the others to continue. (This is how Travis CI works.)

Useful because often things fail for a single Python version (e.g. dependencies not available or syntax change) and seeing it pass for the others helps us understand the problem sooner (than one failing and the rest being cancelled).

```suggestion
      matrix:
        fail-fast: false
```